framework/components/ecl-footers/_footers.scss

```diff
@@ -1,142 +1,144 @@
-/*
-Footer
-
-Site-wide footer component
-
-Markup: html/footer.html
-
-Style guide footer
-*/
+/**
+ * Footers component
+ * @define footers ; weak
+ */
+
+.ecl-footers__site-identity,
+.ecl-footers__site-corporate,
+.ecl-footers__ec {
+  @include ecl-hidden-print();
 
-.footer {
-  background-color: map-get($ecl-colors, 'blue-50');
+  a[#{$ecl-external-link-pattern}]:not(.is-internal):not(#{$ecl-ec-link-patterns})::after {
```

Comment:
I was thinking on this for some time. In direction, do we need to support this at this stage at all, as this is indeed very-specific things carried years for handling very special cases.
Since the scenarios for external, internal, etc. are dark magic to me, in order to avoid breaking some mystical ancient know-how I left it alone, I don't really know how to even test this correctly.